partner_group/views/res_partner_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">Partner Group</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <field name="lang" position="after">
+                <field name="group_id"/>
```

Comment:
This shouldn't be shown following same logic as accounting info (I think `parent_id not set).

Reply:
It has nothing to do with parent_id

Reply:
You are not understanding me: this is a commercial field, and thus, it shouldn't be shown to children partners.